File: pk/fib/fib.py
def fib(n):
    if n == 0: return 0
    elif n == 1: return 1
    else: return fib(n-1) + fib(n-2)

print fib(40)

# python fib.py  48.47s user 0.54s system 96% cpu 50.954 total

File: pk/fib/fib.c
// fib.c
#include <stdio.h>

int main() {
  printf("%d", fib(40));
}

int fib(int n) {
  if(n == 0) {
    return 0;
  } else if(n == 1) {
    return 1;
  }
  return fib(n - 2) + fib(n - 1);
}

// 1.07s user 0.01s system 96% cpu 1.113 total
